src/components/test/Spinner-test.js

```diff
@@ -12,6 +12,11 @@ describe('Spinner', () => {
     assert.isTrue(wrapper.find('SvgIcon.Hyp-Spinner--medium').exists());
   });
 
+  it('uses the registered `hyp-spinner` icon', () => {
+    const wrapper = createSpinner();
+    assert.equal(wrapper.find('SvgIcon').props().name, 'hyp-spinner');
+  });
```

Comment:
We don't typically write manual tests for static markup. In this case you're trying to enforce that components only depend on internally registered icons rather than ones that are exported from the package but not registered internally. That's something that really needs to be solved in a different way.